cmd/mattermost/commands/command.go

```diff
@@ -67,3 +77,41 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func listCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+
+		return err
+	}
+	defer a.Shutdown()
+
+	var teams []*model.Team
+	if len(args) < 1 {
+		teamList, err := a.GetAllTeams()
+		teams = teamList
```

Comment:
It is more idiomatic to move this after the error handling.

Reply:
Correct.